xslt/metadata.php

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
Two issues here:

1. PHP shouldn't be included in this project. The same thing can be accomplished with Python.
2. For this metadata editor, it only has to achieve one thing (for now): translate our generic XML to ISO XML. This doesn't seem to do that.

See this on the standard python library for XSLT transformations: http://lxml.de/xpathxslt.html#xslt